src/otx/core/model/entity/base.py

```diff
@@ -448,3 +449,8 @@ def _read_ptq_config_from_ir(self, ov_model: Model) -> dict[str, Any]:
         initial_ptq_config = argparser.parse_object(initial_ptq_config)
 
         return argparser.instantiate_classes(initial_ptq_config).as_dict()
+
+    @property
+    def model_parameters(self) -> dict:
```

Comment:
I'd rename this to something related to model adapter extra parameters, since this is not related to export